app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = 'User successfully created'
+      redirect_to new_user_path
+    else
+      flash[:error] = 'Something went wrong'
+      render 'new'
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      flash[:success] = 'User was successfully updated'
+      redirect_to new_user_path
```

Comment:
After the user updates their profile, the user should be redirected to `edit_user_path`. They would like to see the update they have made to the profile, right?

- [ ] Please change the redirect path to `edit_user_path`.